src/backend/distributed/master/master_modify_multiple_shards.c

```diff
@@ -162,7 +163,7 @@ LockShardsForModify(List *shardIntervalList)
 	{
 		lockMode = ShareLock;
 	}
-	else if (!HasReplication(shardIntervalList)) /* check if any shards have >1 replica */
+	else if (!HasReplication(shardIntervalList)) /* check if any shard has >1 replicas */
```

Comment:
We don't usually put code comments at the end of the line, should be on the line above.